packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -67,8 +67,19 @@ class _ImageUploadCardState extends State<ImageUploadCard> {
     if (_imageProvider != null) {
       return GestureDetector(
         child: Center(
-            child:
-                Image(image: _imageProvider!, fit: BoxFit.cover, height: 1000)),
+          child: Image(
+            image: _imageProvider!,
+            fit: BoxFit.cover,
+            height: 1000,
+            errorBuilder: (BuildContext context, Object exception,
+                StackTrace? stackTrace) {
+              return Image.asset(
+                  'assets/app/smoothie-icon-transparent.1200x1200.png',
```

Comment:
Could you add a todo that this needs to be changed (or even better change it)

Reply:
With what should I change it? I can't use any images from internet?

Reply:
What about using something like this https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Awifi_off%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048